airbyte-metrics/lib/src/main/java/io/airbyte/metrics/lib/AirbyteMetricsRegistry.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.metrics.lib;
+
+import com.google.api.client.util.Preconditions;
+
+/**
+ * Enum source of truth of all Airbyte metrics. Each enum value represent a metric and is linked to
+ * an application and contains a description to make it easier to understand.
+ *
+ * Each object of the enum actually represent a metric, so the Registry name is misleading. The
```

Comment:
i'm not sure i understand why the name is misleading. i think it makes sense to me.

Reply:
great! looks like I was overly concerned then.